website/docs/configuration/observability.md

```diff
@@ -36,10 +36,39 @@ $ docker logs my-ferretdb
 
 FerretDB writes logs to the standard error (`stderr`) stream.
 
-## Metrics
+## Debug handler
 
-FerretDB exposes metrics in Prometheus format on the debug handler on `http://127.0.0.1:8088/debug/metrics` by default.
-There is no need to use an external exporter.
+FerretDB exposes various HTTP endpoints with the debug handler on `http://127.0.0.1:8088/debug/` by default.
 The host and port can be changed with [`--debug-addr` flag](flags.md#interfaces).
 
-Please note that the set of metrics is not stable yet; metric and label names and formatting of values might change in minor releases.
+### Metrics
+
+FerretDB exposes metrics in Prometheus format on the `/debug/metrics` endpoint.
+There is no need to use an external exporter.
+
+<!-- https://github.com/FerretDB/FerretDB/issues/3420 -->
+
+:::note
+The set of metrics is not stable yet; metric and label names and value formatting might change in minor releases.
+:::
+
+### Probes
+
+FerretDB exposes the following probes that can be used for
+[Kubernetes health checks](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
+and similar use cases.
+They return HTTP code 2xx if a probe is successful and 5xx otherwise.
```

Comment:
Will there be fixed HTTP codes for this? e.g. `200 or 500`

Reply:
https://github.com/FerretDB/FerretDB/pull/4424#discussion_r1666007724